www/Dockerfile

```diff
@@ -1,9 +1,13 @@
 FROM alpine AS oqs-provider
 
-RUN apk add --no-cache cmake git g++ make ninja openssl openssl-dev
```

Comment:
This doesn't look right (`docker run --rm alpine cmake` says `cmake` isn't found)

Reply:
`docker run --rm alpine apk add cmake` shows successful installation for me

Reply:
yes, but you're removing this line!

Reply:
You're right! I got confused by Github's green check mark.